include/swift/Reflection/ReflectionContext.h

```diff
@@ -18,6 +18,10 @@
 #ifndef SWIFT_REFLECTION_REFLECTIONCONTEXT_H
 #define SWIFT_REFLECTION_REFLECTIONCONTEXT_H
 
+#ifdef __APPLE__
```

Comment:
Would you mind switching this to:

    #if defined(__APPLE__) && defined(__MACH__)

Reply:
I shall!